frontends/main/src/app-pages/DepartmentListingPage/DepartmentListingPage.tsx

```diff
@@ -211,6 +210,7 @@ const DepartmentListingPage: React.FC = () => {
             current="Departments"
           />
         }
+        backgroundUrl={backgroundSteps.src}
```

Comment:
The `<Banner>` component has a default image path (background_steps.jpg), but as this is just a path string, the image is loaded directly as opposed to from the image optimization endpoint. Static imports get the image optimzation endpoint, but as a rule the `<Banner>` component cannot import from outside ol-components.